src/wallet/coincontrol.h

```diff
@@ -33,11 +33,10 @@ class CCoinControl
     CTxDestination destChange = CNoDestination();
     //! Override the default change type if set, ignored if destChange is set
     std::optional<OutputType> m_change_type;
-    //! If false, only selected inputs are used
-    bool m_add_inputs = true;
     //! If false, only safe inputs will be used
     bool m_include_unsafe_inputs = false;
-    //! If false, allows unselected inputs, but requires all selected inputs be used
+    //! If true, the selection process can add extra unselected inputs from the wallet
+    //! while requires all selected inputs be used
     bool fAllowOtherInputs = false;
```

Comment:
nit: It would be nice if we could switch to using the new naming scheme. Perhaps a scripted diff?

Reply:
pushed